scripts/qa_crowbarsetup.sh

```diff
@@ -3261,23 +3228,15 @@ function set_proposalvars
         deployceph=
     fi
     # C3: Cloud5 only has ceph for SLES12
-    if iscloudver 5 && [ -z "$want_sles12" ] ; then
+    if iscloudver 5 ; then
```

Comment:
but thats the intention, right? see comment: 

# C3: Cloud5 only has ceph for SLES12